README.rst

```diff
@@ -1 +1,94 @@
-A tool to convert numbers in natural language to decimal numbers.
\ No newline at end of file
+=============
+number-parser
+=============
+.. image:: https://img.shields.io/pypi/pyversions/price-parser.svg
+   :target: https://pypi.python.org/pypi/price-parser
+   :alt: Supported Python Versions
+
+``number-parser`` is a simple library that allows you to convert numbers written in the natural
+language to it's equivalent numeric forms. It currently supports cardinal numbers in the following 
+languages - English, Hindi, Spanish and Russian.
+
+Installation
+============
+::
+
+    pip install number-parser
+
+number-parser requires Python 3.6+.
+
+Usage
+=====
+
+The library provides two major APIs which corresponds to the following two common usages.
+
+1. Identifying the numbers in a text string, converting them to corresponding numeric values while ignoring non-numeric words.
+2. Converting a single number written in words to it's corresponding integer. 
+ 
+
+Interface #1: Multiple numbers 
+--------------------------------
+
+>>> from number_parser import parse
+>>> parse("I have two hats and thirty seven coats")
+'I have 2 hats and 37 coats'
+>>> parse("One, Two, Three go")
+'1, 2, 3 go'
+
+
+Interface #2: Single number 
+--------------------------------
+
+>>> from number_parser import parse_number
+>>> parse_number("two thousand and twenty")
+2020
+>>> output = parse_number("not_a_number")
+>>> output
+None
+
+
+Language Support
+----------------
+
+The default language is English, you can pass the language parameter with corresponding locale for other languages.
+
+>>> from number_parser import parse, parse_number
+>>> parse("Hay tres gallinas y veintitrés patos", language = 'es')
```

Comment:
Itis a good idea to follow PEP8 when using examples. To do it, you should change those cases where there is a space between "language" and the string (example: `language = 'es'` -->  `language='es'`